src/App.tsx

```diff
@@ -1,109 +1,36 @@
 import React from 'react';
-
 import './App.scss';
-
-// import postsFromServer from './api/posts';
-// import commentsFromServer from './api/comments';
-// import usersFromServer from './api/users';
+import { Post } from './types/Post';
+import { User } from './types/User';
+import { Comment } from './types/Comment';
+import { PreparedPost } from './types/PreparedPost';
+import postsFromServer from './api/posts';
+import commentsFromServer from './api/comments';
+import usersFromServer from './api/users';
+import { PostList } from './components/PostList';
+
+function findUser(users: User[], post: Post): User | null {
```

Comment:
![image](https://user-images.githubusercontent.com/91411692/232747564-9ddb7eb8-ea6b-4346-bd64-9beab035f11e.png)
```suggestion
function getUserById(users: User[], id: number): User | null {
```